resources/cassandra/INSTALL.md

```diff
@@ -0,0 +1,57 @@
+# Prerequisites
+The cassandra exporter generates a high number of metrics as the Sysdig Agent has a limit of timeseries that can send to Sysdig Monitor, you have to deploy a Prometheus server and create the recording rules that we provide. This way, we will filter only the metrics that we need.
+
+To deploy a Prometheus server you will need:
+* [helm](https://helm.sh/docs/intro/install/)  
+* [helmfile](https://github.com/roboll/helmfile)
+
+# Installing the exporter
+Cassandra expose the metrics with jmx, the exporter gather this metrics and expose them with a prometheus format, usually the jmx is unsecured and it has not user or password, in this case the best way to do it is to use a sidecar with the exporter.
```

Comment:
Cassandra expose the metrics with jmx and the exporter gather this metrics and expose them in Prometheus format. Usually jmx is unsecured and it has not user or password. In this case, the best way to deploy it is to add a sidecar with the exporter.